style.css

```diff
@@ -520,15 +545,12 @@ option {
     padding: 0;
     width: 100%;
   }
-  .section-form{
-    height: 150rem;
-  }
   form{
     width: 100%;
     margin-bottom: 5rem;
   }
   input{
-    font-size: 4rem;
+    font-size: 1.6rem;
```

Comment:
Is this required to work position the header button?

Reply:
It was some work I did before the ticket came through

Reply:
@TurnbulA As discussed, please only commit one feature per PR.